container/image.bzl

```diff
@@ -551,6 +561,8 @@ _attrs = dicts.add(_layer.attrs, {
     # We need these flags to distinguish them.
     "null_cmd": attr.bool(default = False),
     "null_entrypoint": attr.bool(default = False),
+    "operating_system": attr.string(default = "linux"),
```

Comment:
This was already being inherited from [here](https://github.com/bazelbuild/rules_docker/blob/834fa017067752b5d7bd908bccbddffbd6687f0b/container/layer.bzl#L250) because of the `_layer.attrs` above.

Reply:
Done!